src/index.ts

```diff
@@ -47,6 +47,16 @@ export interface QueryResult {
   metadata?: ColumnMetadata[];
 }
 
+/**
+ * Possible result for any attach or dettach database command
+ * status: 0 or undefined for correct execution, 1 for error
+ *  message: if status === 1, here you will find error description
+ */
+export interface AttachDetachResult {
```

Comment:
I'm starting to get a feeling there are too many entities that are similar, maybe there should be a base interface that has `status` and `message`

Reply:
I completely agree.
Base result interface created and now other result types are extending it.